app/src/main/res/layout/fragment_main.xml

```diff
@@ -80,12 +80,19 @@
         android:orientation="vertical"
         android:layout_below="@id/mainBalanceLayout">
 
-        <android.support.v7.widget.RecyclerView
-            android:id="@+id/mainTransactionList"
+        <android.support.v4.widget.SwipeRefreshLayout
+            android:id="@+id/mainTransactionListSwipe"
```

Comment:
Theres no need to replace the id to include Swipe, its still the main transaction list UI element so the old name is good.